RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# Pinny Release Checklist
+
+Take this checklist and paste it into your release PR. Ensure steps are followed in order.
+
+- [ ] Tests pass in the terminal and in supported browsers.
+  - See [README](./README.md#browser-compatibility) for list of supported browsers.
```

Comment:
You can write automated tests for those, can't you? Shouldn't automation be reached for first, and only then extend this with a list of non-testable regressions? Would adding that section make people lazy in that they won't bother trying to automate it?